samples/nextjs/package.json

```diff
@@ -31,8 +31,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sitecore-jss/sitecore-jss-nextjs": "^15.0.0",
-    "@sitecore-jss/sitecore-jss-tracking": "^15.0.0",
+    "@sitecore-jss/sitecore-jss-nextjs": "*",
```

Comment:
I think we should just keep this in sync with the rest ("^15.0.1"). Now that the nextjs package and sample are merged into dev, they should be kept in sync when we bump version. Also, we don't have to remember to remove the asterisk when we release :)